backend/src/main/java/ddangkong/domain/balance/content/BalanceContentRepository.java

```diff
@@ -1,13 +1,33 @@
 package ddangkong.domain.balance.content;
 
-import ddangkong.domain.balance.option.BalanceOption;
+import static java.util.stream.Collectors.toList;
+
 import ddangkong.exception.BadRequestException;
+import ddangkong.exception.InternalServerException;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.LongStream;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface BalanceContentRepository extends JpaRepository<BalanceContent, Long> {
 
+    long count();
+
+    List<BalanceContent> findByIdIn(List<Long> ids);
+
     default BalanceContent getById(Long id) {
         return findById(id)
                 .orElseThrow(() -> new BadRequestException("해당 질문 컨텐츠가 존재하지 않습니다."));
     }
+
+    default List<BalanceContent> findByRandom(int count) {
+        List<Long> ids = LongStream.rangeClosed(1, count).boxed().collect(toList());
+        if (ids.size() < count) {
+            throw new InternalServerException("질문이 라운드만큼 존재하지 않습니다.");
+        }
+
+        Collections.shuffle(ids);
+        List<Long> candidateIds = ids.subList(0, count);
+        return findByIdIn(candidateIds);
+    }
```

Comment:
의견) 이러면 1부터 totalRound 개수 만큼만 셔플이 될 것 같아요! 
다음과 같이 수정해주면 좋을 것 같아요!
추가적으로 findByIdIn 이 자동으로 오름차순으로 정렬되는 것 같아서 이 결과도 셔플해주면 더 좋지 않을까 생각해서 추가해봤어요!

```suggestion
    default List<BalanceContent> findByRandom(int count) {
        List<Long> ids = LongStream.rangeClosed(1, count()).boxed().collect(toList());
        if (ids.size() < count) {
            throw new InternalServerException("질문이 라운드만큼 존재하지 않습니다.");
        }

        Collections.shuffle(ids);
        List<Long> candidateIds = ids.subList(0, count);
        List<BalanceContent> result = findByIdIn(candidateIds);
        Collections.shuffle(result);
        return result;
    }
```

Reply:
오! 타칸 놓친 부분 알려줘서 고마워요~ 리뷰한 것 바탕으로 수정해볼꼐요~